features/step_definitions/web_steps.rb

```diff
@@ -162,6 +162,13 @@ def with_scope(locator)
   end
 end
 
+Then /^I should have (\d+) new category$/ do |count|
+  # pending # express the regexp above with the code you wish you had
```

Comment:
The `pending` should be reserved for scenarios that you are still implementing, if you are done, you should remove this line